src/cmap/connection.ts

```diff
@@ -384,7 +384,8 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
       } else {
         // Get the first orphaned operation description.
         const entry = this[kQueue].entries().next();
-        if (entry) {
+        /* eslint no-restricted-syntax: 0 */
```

Comment:
Eslint wants us to check for falsey here but we explicitly want the check to be for `undefined`.

Reply:
eslint wants you to use `entry.value != null` (double, not triple eq) which should cover what we're trying to gate here, but its true this will only be undefined explicitly so I don't have a strong onion otherwise